### PR TITLE
Made IRC optional, added a timeout and some error messages

### PR DESCRIPTION
**Pull request by Victorious3**

### CODE REVIEW

File: commands.ls [hunk 1]
```diff
@@ -32,7 +32,7 @@ context =
       "Please, say my name again." "Talk to me more."
       "I'm here for you." "Don't stop." "Continue."
       "Please ask copygirl to add more responses."
-
```
Comment (copygirl) [inline commands.ls:35]: The indentation here and other empty lines should stay.

File: cord.ls [hunk 2]
```diff
@@ -4,27 +4,42 @@ require! {
...
+for plugin in plugins
+  (require plugin) discord, irc
+
+# Connect to discord
+discord.login ...auth.discord<[email password]>, (err) !-> if err?
+  console.error "[Discord] #err"
+  finalize!
+
+# Quit nicely when pressing CTRL-C
+process.on \SIGINT, !-> finalize!
```
Comment (copygirl) [inline cord.ls]: `process.on \SIGINT, finalize` would work as well.

File: cord.ls [hunk 3]
```diff
@@ -4,27 +4,42 @@ require! {
...
+else console.log "[= IRC =] No channel selected!"
+
+discord = new DiscordClient!
+
+# Loading modules, irc parameter is optional, may be undefined
+for plugin in plugins
+  (require plugin) discord, irc
+
+# Connect to discord
+discord.login ...auth.discord<[email password]>, (err) !-> if err?
```
Comment (copygirl) [inline cord.ls]: Since discord-js supports Promises, might be cleaner to use those instead of callbacks.

**Comment by Victorious3** [comment]
This should be done now

**Pull request merged**